channeldb/db.go

```diff
@@ -520,7 +520,7 @@ func fetchChannels(d *DB, pending, waitingClose bool) ([]*OpenChannel, error) {
 					// than Default, then it means it is
 					// waiting to be closed.
 					channelWaitingClose :=
-						channel.ChanStatus() != Default
+						channel.ChanStatus() != ChanStatusDefault
```

Comment:
Arguably, it _is_ as we can't do anything with it before the remote party force closes. This section was modified slightly, but I ended up introducing a bug the first time around. I have a plan to make a follow up PR to extend `listchannels` to also display the full string chan status as well. This will help users to identify which channels have been restored and are still waiting for the recovery process to kick ouff.